.github/workflows/ci_on_scratch.yml

```diff
@@ -0,0 +1,29 @@
+name: Normal CI
+
+on:
+  push:
+
+jobs:
+  run-test:
+    name: Run Test on Scratch
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: Install yarn
+        working-directory: ./frontend
+        run: npm install -g yarn
+      - name: Install yarn packages
+        working-directory: ./frontend
+        run: yarn install
```

Comment:
```suggestion
        run: yarn install --frozen-lockfile
```

Reply:
In CI, can you add `--frozen-lockfile` option?
https://qiita.com/akameco/items/8945e434da315a0fbe44